twenty-first/src/math/bfield_codec.rs

```diff
@@ -484,9 +522,11 @@ mod tests {
     use proptest_arbitrary_interop::arb;
     use test_strategy::proptest;
 
+    use crate::bfe;
     use crate::prelude::Digest;
     use crate::prelude::Tip5;
     use crate::prelude::XFieldElement;
+    use crate::xfe;
```

Comment:
This might be the time to just include the entire prelude.

```suggestion
    use crate::prelude::*;
```